apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -342,6 +338,15 @@ private Coverage transformPartC(Beneficiary beneficiary, Profile profile) {
       transformEntitlementDualEligibility(coverage, beneficiary);
     }
 
+    String ContractAndPbpID = concatenatePartCContractAndPbp(beneficiary);
```

Comment:
It looks like the changes here will affect every response, but we only want to make these changes for the C4DIC profile (`profile.equals(Profile.C4DIC))`)

Reply:
Good Catch